src/testing/reduced_order.h

```diff
@@ -4,20 +4,36 @@
 #include "tests.h"
 #include "dg/dg.h"
 #include "parameters/all_parameters.h"
+#include "pod_adaptive_sampling.h"
+#include "functional/functional.h"
+#include "functional/lift_drag.hpp"
+#include "parameters/all_parameters.h"
+#include <deal.II/numerics/solution_transfer.h>
+#include "flow_solver/flow_solver.h"
+#include "flow_solver/flow_solver_factory.h"
+#include <deal.II/base/numbers.h>
+#include "dg/dg_factory.hpp"
+#include "ode_solver/ode_solver_factory.h"
+#include <iostream>
+#include "functional/functional.h"
```

Comment:
by including `flow_solver.h`, some of these includes aren't needed like dg and ode. Also, include only things that the .h file needs here, and the .cpp ones should be moved to the .cpp file; this might explain it better: https://stackoverflow.com/a/15420975

Reply:
Fixed in most recent commit.